.github/CODEOWNERS

```diff
@@ -31,7 +31,7 @@
 /avm/res/compute/virtual-machine/ @Azure/avm-res-compute-virtualmachine-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/compute/virtual-machine-scale-set/ @Azure/avm-res-compute-virtualmachinescaleset-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/consumption/budget/ @Azure/avm-res-consumption-budget-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/container-instance/container-group/ @Azure/avm-res-containerinstance-containergroup-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/container-instance/container-group/ @Azure/avm-res-containerinstance-containergroup-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Note, the team is either not yet existing or parent not yet approved

Reply:
@JPEasier - Please create the Github Team required for the module

Reply:
https://github.com/orgs/Azure/teams/avm-res-containerinstance-containergroup-module-owners-bicep

Reply:
https://github.com/orgs/Azure/teams/avm-res-containerinstance-containergroup-module-contributors-bicep